### PR TITLE
Quiz result column design fix

### PR DESCRIPTION
**Pull request by dvira**
image

### CODE REVIEW

File: src/components/Screen-v2/UsersScreen/style.module.scss [hunk 1]
```diff
@@ -57,6 +57,18 @@
...
+
+    .failure {
+      color: var(--red);
+    }
+
+    .success {
+      color: var(--green);
+    }
+
+    .none {
```
Comment (dr15) [inline src/components/Screen-v2/UsersScreen/style.module.scss:69]: It's not quite clear what "none" means, does it mean no data? I would change the name

**Review by dr15 — APPROVED** [review]
Approved but left a a comment

**Review by RonnyO — APPROVED** [review]

**Comment by RonnyO** [comment]
@dvira fine with me, just address Denise's comment and you're good to go

File: src/components/Screen-v2/UsersScreen/style.module.scss (hunk 1 above)
Comment (dvira) [reply src/components/Screen-v2/UsersScreen/style.module.scss]: Done.

**Pull request merged**